core/unit_test/incremental/Test12b_TeamScratch.hpp

```diff
@@ -105,9 +107,17 @@ struct TeamScratch {
 
 TEST(TEST_CATEGORY, IncrTest_12b_TeamScratch) {
   TeamScratch<TEST_EXECSPACE> test;
+  // Fails with larger vector sizes and if the team-size is not a multiple
+  // of 32.
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
```

Comment:
Ok as a temporal workaround. Has the issue of failing with non-multiples of 32 been communicating to the CLANG folks?

Reply:
I am not sure if this has been communicated to the clang compiler developers yet. I am in the process of writing a small reproducer for this issue to be submitted as a bug. Hopefully this is something that is easily fixable.